lib/PayPal/Core/PayPalHttpConnection.php

```diff
@@ -93,6 +93,7 @@ public function execute($data)
         switch ($this->httpConfig->getMethod()) {
             case 'POST':
                 curl_setopt($ch, CURLOPT_POST, true);
+                break;
```

Comment:
If so, there is a logical problem I think. 
if method is POST then
```
curl_setopt($ch, CURLOPT_POST, true);
curl_setopt($ch, CURLOPT_POSTFIELDS, $data);
```
and if method is one of [POST, PUT, PATCH, DELETE] then

```
curl_setopt($ch, CURLOPT_POSTFIELDS, $data);
```

am I right?